gtda/diagrams/tests/test_features.py

```diff
@@ -85,11 +85,11 @@ def test_pi_positive(pts):
 
 
 def _validate_distinct(X):
-    unique_values = [np.unique(x[0:2, :]) for x in X]
+    """Check if, in X, there is any persistence diagram for which all births
+    and deaths are equal."""
+    unique_values = [np.unique(x[:, 0:2]) for x in X]
     if np.any([len(u) < 2 for u in unique_values]):
-        raise ValueError("There should be at least two distinct points"
-                         "in the persistent diagrams:" +
-                         "now, only {} is present".format(*unique_values))
+        raise ValueError
```

Comment:
I am surprised that you  do  not need parentheses  after `ValueError`.

Reply:
https://docs.python.org/3/tutorial/errors.html#raising-exceptions